.circleci/config.yml

```diff
@@ -91,23 +91,28 @@ jobs:
             cf login -a https://api.fr.cloud.gov -u "$CF_USER" -p "$CF_PASSWORD" -o "$CF_ORG"  -s "$CF_SPACE"
             cf ssh cms --command "/var/www/vendor/bin/drush state:set system.maintenance_mode 0 && \
                                   /var/www/vendor/bin/drush pm:uninstall usagov_login && \
+                                  /var/www/vendor/bin/drush pm:enable usagov_benefit_finder_block && \
```

Comment:
We have removed usagov_benefit_finder_block module

Reply:
I made the suggested change. Please review again.